filters/builtin/stripquery_test.go

```diff
@@ -91,3 +92,21 @@ func TestPreserveQuery(t *testing.T) {
 		}
 	}
 }
+
+func BenchmarkStripQuery(b *testing.B) {
+	piece := "query=cXVlcnkgewogIG5hbWUKfQo%253D&variables=&apiEndpoint=https%253A%252F%252Fwww.zalando.de%252Fapi%252Fgraphql%252Fpreview&frontendType=web&zalandoFeature=manual"
+	q := strings.Repeat(piece, 1000)
```

Comment:
Is 1000 realistic? There is a limit to the length of query string anyways, I think we should use something more realistic (say 2x size of max length you see in your system)

Reply:
We also prefer to not put system-specific values into tests, you can use https://en.wikipedia.org/wiki/Metasyntactic_variable and `xxxxxx` for names or values as this does not affect the performance.